live-examples/js-examples/intl/intl-locale.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">const korean = new Intl.Locale("ko", {script: "Kore", region: "KR", hourCycle: "h24", calendar: "gregory"});
```

Comment:
Rather than have it on one line it would be easier to read like:

```js
const korean = new Intl.Locale("ko", {
  script: "Kore", region: "KR", hourCycle: "h24", calendar: "gregory"
});
```

Reply:
Also here and elsewhere, single quotes for strings is our style.
(although note: use double for the expected output strings)